src/classes/StorageUtility.ts

```diff
@@ -0,0 +1,25 @@
+export class StorageUtility {
```

Comment:
ℹ️ Saw these referenced in a StackOverflow post as the quick & easy way modernizr does it